benchmarks/dynamo/huggingface.py

```diff
@@ -404,12 +395,31 @@ def load_model(
         else:
             config, model_cls = EXTRA_MODELS[model_name]
 
+        return model_cls, config
+
+    @download_retry_decorator
+    def _download_model(self, model_name):
```

Comment:
Based on the stack dump at https://hud.pytorch.org/pytorch/pytorch/commit/862d658059e6006a0d07afd9a2672ea647d3865f, I don't think this is where pretrained model is downloaded?

Reply:
It seems like sometimes we just grab the class and instantiate, and other times we use AutoConfig to get the model. I'll setup the retry to capture both cases.